target/linux/ath79/dts/qca9557_mikrotik_routerboard-960pgs.dts

```diff
@@ -0,0 +1,310 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-960pgs", "qca,qca9557";
+	model = "MikroTik RouterBOARD 960PGS";
+
+	aliases {
+		serial0 = &uart;
```

Comment:
Missing led aliases

Reply:
I did miss led aliases completely. I'll add following aliases in next push
led-boot = &led_user;
led-failsafe = &led_user;
led-running = &led_user;
led-upgrade = &led_user;